src/style/global-style.ts

```diff
@@ -6,6 +6,12 @@ import {
 import { baseTheme } from "./themes";
 
 const GlobalStyle = createGlobalStyle`
+
+ :root {
```

Comment:
Comment: if we're going to add this as a temporary token I think it's better added to the ScopedTokenProvider rather than global styles to maintain the separation of concerns